### PR TITLE
FR-13416 - Trigger e2e test for alpha versions

### PR DESCRIPTION
**Pull request by frontegg-david**

### CODE REVIEW

File: .github/workflows/publish-prerelease.yml [hunk 1]
```diff
@@ -103,6 +103,44 @@ jobs:
...
+
+      - name: Trigger E2E test workflow
+        uses: actions/github-script@v5
+        env:
+          PR_REF: '${{ steps.cpr.outputs.pull-request-head-sha }}'
+          PR_VERSION: '${{ steps.publish_pre_release_version.outputs.LIB_VERSION }}-alpha.${{ github.run_id }}'
+        with:
+          github-token: ${{ secrets.GH_REPOSITORY_ADMIN_TOKEN }}
+          script: |
+            const sha = process.env.PR_REF;
```
Comment (guy-frontegg) [inline .github/workflows/publish-prerelease.yml:129]: Why are we doing the dispatch like this via code?

File: .github/workflows/trigger-e2e-test.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          github-token: ${{ secrets.GH_REPOSITORY_ADMIN_TOKEN }}
+          script: |
+            const {repo, sha, version} = process.env;
+            const owner = context.payload.repository.organization
+            const e2eRepo = 'e2e-system-tests'
+            const workflow_id = 'frontegg-react-e2e-tests.yml'
+            const context = `${owner}/${e2eRepo}`
+            const dispatch_id = `${repo}/${sha}`
+
+            const res = await github.rest.repos.createCommitStatus({
```
Comment (guy-frontegg) [inline .github/workflows/trigger-e2e-test.yml:33]: There are actions to do it

**Review by rotemzif1 — APPROVED** [review]

File: .github/workflows/publish-prerelease.yml (hunk 1 above)
Comment (frontegg-david) [reply .github/workflows/publish-prerelease.yml:129]: I just used the official github action script code, I need the PR sha so I can prevent from merging before the "private workflow" finished

File: .github/workflows/trigger-e2e-test.yml (hunk 2 above)
Comment (frontegg-david) [reply .github/workflows/trigger-e2e-test.yml:33]: After we finish building our Frontegg CI App, this code will be replaced with more elegant way

**Pull request merged**